### PR TITLE
Add docs

### LINKED ISSUE #13: Howto for notifications
Hello can you please make a howto showing how to make push notification when washing/dryer machine is finished it sends out a notification to all devices like a iPhone

And also a howto so when it is finished it sends a command to my speakers to say washing machine is now finessed 

Make the howto so it’s made for beginners

### LINKED ISSUE #22: Washing machine timer
Hi ollo69,

First of all, this is awsome, you are awsome!

I was just woundering as you are the expert, is it easy to make a countdown timer of % progress bar for a wash that is running?

Like the "initial_time" - "remain_time" = % done or a timer?
Now I only have the binary sensor which is also handy but a timer would be even more awsome.

I have seen a few solutions but I think you would probably have a better solution :)

### LINKED ISSUE #39: How to use the binary_sensor.lavatrice_wash_completed
Hi,
I have an automation in my setup that listen for changes of the binary_sensor.lavatrice_wash_completed, but seems this never happens. 
Also in history, the state is never changed. 
The automation works, if i trigger manually the action is fired. 
Can you tell me how to catch the washing complete event correctly?
Thanks

### LINKED ISSUE #50: [Feature Request] lovelace card
Can someone make a lovelace card for washing machine and dryer machine?

### LINKED ISSUE #62: What are all of these attrs for the washer?
I can guess some of them, but can someone please give me what they would actually be? (Don't want to waste water by running the washer!)

- [x] model: T1872EFHU_F-TL **Where does this come from?**
- [x] mac_address: 74:40:be:c8:5b:41 **What kind of MAC address? Bluetooth/Wifi?**
- [x] run_completed: off **Turns to `on` when completed**
- [x] error_state: off **Turns to `on` when error**
- [ ] error_message: - **Is this a word or a sentence or a number or what?**
- [x] run_state: - **Current state in words**
- [x] pre_state: Standby **Previous state in words**
- [ ] current_course: - **Is this a word or a sentence or a number or what?**
- [ ] spin_option_state: No Spin **What are the different options?**
- [ ] watertemp_option_state: - **Is this a word or a sentence or a number or what?**
- [ ] drylevel_option_state: - **Is this a word or a sentence or a number or what?**
- [ ] tubclean_count: 4 **Time until tubcleans, tubcleans completed, washes until tubcleans, washes since last tubclean?**
- [x] remain_time: 0:00 **Hour:Minute time remaining on cycle**
- [x] initial_time: 0:00 **Hour:Minute initial time on cycle**
- [ ] reserve_time: 0:00 **Is this a word or a sentence or a number or what?**
- [ ] doorlock_mode: Off **What does this mean?**
- [ ] doorclose_mode: - **What does this mean?**
- [ ] childlock_mode: Off **What does this mean?**
- [x] remotestart_mode: Off **Turns on when you can control it, right?**
- [ ] creasecare_mode: - **???**
- [ ] steam_mode: Off **???**
- [ ] steam_softener_mode: - **???**
- [ ] prewash_mode: Off **???**
- [x] turbowash_mode: Off **Turns on when Turbowash is selected**
- [x] friendly_name: Wetty **Name in ThinQ**
- [x] icon: mdi:washing-machine **Icon for device**

### PR DESCRIPTION
**Pull request by KTibow**
Just add a bit of docs. (I'm fine if you request/comment on changes or for changes!)
Closes #22
Closes #13
Fixes #39
Fixes #50
Mostly fixes #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,90 @@ Edit it by adding the following lines:
...
+    from: 'off'
+    to: 'on'
+  condition: []
+  action:
+  - data:
+      title: 'The clothes are dry!'
+      message: 'Get them while they're hot!'
+    service: notify.notify
+```
+- Washer status card (LG전자 / CC BY (https://creativecommons.org/licenses/by/2.0) for image. Find the images [here](/washerpics/))
```
Comment (KTibow) [inline README.md:135]: Can someone make sure that this works?

File: README.md [hunk 2]
```diff
@@ -82,6 +82,90 @@ Edit it by adding the following lines:
...
+    conditions:
+      - entity: sensor.my_washer
+        not_state: "on"
+    card:
+      aspect_ratio: '1'
+      entity: sensor.my_washer
+      image: /local/washer.jpg
+      type: picture-entity
+type: vertical-stack
+```
```
Comment (KTibow) [inline README.md:168]: Can someone make sure that this card works?

Comment (KTibow) [reply README.md:249]: Never mind, everything is tested (except the automation).

**Comment by KTibow** [comment]
@ollo69 done with everything, ready for review 🎉!

**Comment by KTibow** [comment]
@ollo69 Finished up, what do you think of the docs?

**Comment by ollo69** [comment]
Thanks again for your great job and contribute on repository, really appreciated.
I will take same time later or tomorrow to merge your PR, on this days I'm really busy at work.

**Comment by KTibow** [comment]
Got it, sounds good. I'm working on the laundry card a bit more, so not ready yet...

**Pull request merged**